app/models/grda_warehouse/remote_credentials/s3.rb

```diff
@@ -10,5 +10,22 @@ module GrdaWarehouse
   class RemoteCredentials::S3 < GrdaWarehouse::RemoteCredential
     alias_attribute :s3_access_key_id, :username
     alias_attribute :s3_secret_access_key, :password
+    alias_attribute :s3_prefix, :path
+
+    def s3
+      @s3 ||= if s3_secret_access_key.present? && s3_secret_access_key != 'unknown'
+        AwsS3.new(
+          region: region,
+          bucket_name: bucket,
+          access_key_id: s3_access_key_id,
+          secret_access_key: s3_secret_access_key,
```

Comment:
I don't know if this is correct. It doesn't work locally fo rme to use the `password` or `encrypted_password`. I copied this logic from below file, which seems to be doing some encryption too.

https://github.com/greenriver/hmis-warehouse/blob/6700d366d37a29903555ab9194f88f28ac6af2f5/app/models/grda_warehouse/hmis_import_config.rb#L9

Reply:
That part makes sense, but `s3_secret_access_key` seems to return the unencrypted password. When comparing to `HmisImportConfig`, I'm not sure if the attr_encrypted `s3_secret_access_key` returns an encrypted or decrypted access key. I would guess that `AwsS3` expects an unencrypted key, which is what I'm passing

Reply:
yes, `s3_secret_access_key` should return the unencrypted value.  `AwsS3` definitely wants the unencrypted value.  `attr_encrypted` is the gem that makes that magic happen.